### ISSUE TITLE
You should change simple demo.

### ISSUE THREAD

**Issue by tomo120717** [lucys0/awe#8]
Hello,

Thank you for codes.
I could not conduct your next codes.

```
pip install waypoint-extraction
import waypoint_extraction as awe
waypoints = awe.extract_waypoints(states, err_threshold)
```

I found this page https://pypi.org/project/waypoint-extraction/#description to install waypoint-extraction.
This page's instruction and upper one are different.

```
pip install robot-awe
from robot_awe import extract_waypoints
waypoints = extract_waypoints(states, err_threshold)
```
You should change this page one.

Thank you.

**Comment by henry0922** [comment]
Indeed, that is the case. The user manual is not informative and provides no reference value, as it fails to explain what the given parameters are.

**Comment by dazhongtu** [comment]
Hello, did you find reference value? What should 'states' be like in `waypoints = extract_waypoints(states, err_threshold)`
